queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java

```diff
@@ -288,4 +300,22 @@ private void sleepSporadically(final long loopTimeMsec) throws InterruptedExcept
         }
 
     }
+
+
+    private interface RunnableRawCallback {
+        void callback() throws InterruptedException;
+    }
+
+    private void withHandlingRuntimeException(final RunnableRawCallback cb) throws InterruptedException {
+        try {
+            cb.callback();
+        } catch (final DBIException e) {
+            log.warn(String.format("%s: Thread %s got DBIException exception: %s",
```

Comment:
![image](https://user-images.githubusercontent.com/51940/65898867-60d49200-e3aa-11e9-9c98-7cb1b69bb698.png)

https://www.slf4j.org/faq.html#logging_performance 😏

Reply:
Ok, actually, now i remember.. most logs in the `queue` code are written this way:
* Should i only update this new one ?
* Should i  update the whole file?
* Should i  update the whole queue code base?
* Should i  update nothing ?

Reply:
Up to you. FWIW my vote would be to update the current file. 👼 

> Leave Things BETTER than you found them. ~ Robert Baden Powell

Reply:
> FWIW my vote would be to update the current file. 👼

👍 

> Robert Baden Powell 

Really, who is that Dude anyways :-)

Reply:
@pierre see cb6d28806243f280ee6